src/poggit/release/details/review/ReviewAdminAjax.php

```diff
@@ -72,8 +72,16 @@ protected function impl() {
 
                 if(!Meta::isDebug()) {
                     $ip = Meta::getClientIP();
-		    $clean = str_replace("`", "'", $message);
-                    Discord::reviewsHook("{$session->getName()} reviewed https://poggit.pmmp.io/p/{$repoIdRows[0]["name"]}/{$repoIdRows[0]["version"]} ($score/5):\n\n```\n$clean\n```", "User reviews");
+		    		$clean = str_replace("`", "'", $message);
```

Comment:
You are using tabs instead of spaces here.

That aside... I don't think poggit has proper indentation anyway... HTML is 2 spaces and PHP is 4 spaces and HTML in PHP is 4+2 spaces...